### DISCUSSION TITLE
Using local version of ansible with vagrant

### DISCUSSION THREAD

**Discussion by tosinosu** [ansible-community/molecule-plugins#259] [vagrant] [answered]
In vagrantfile, it is possible to run all playbooks on the guest machine, using ansible local like below

`config.vm.provision "ansible_local"` 

Is it possible to achieve the same purpose with vagrant driver in molecule? If yes, how?

**Comment by konstruktoid** [comment, marked as answer]
Hi @tosinosu,
no it isn't at the moment, the molecule vagrant driver uses `ssh`.

**Reply by tosinosu** [reply]
Thanks for your response @konstruktoid
